homeassistant/components/homekit/type_covers.py

```diff
@@ -45,6 +45,13 @@ class GarageDoorOpener(HomeAccessory):
     and support no more than open, close, and stop.
     """
 
+    HASS_TO_HK_STATE = {
+        STATE_OPEN: 0,
+        STATE_CLOSED: 1,
+        STATE_OPENING: 2,
+        STATE_CLOSING: 3,
+    }
```

Comment:
Just compared this to other files of homekit/type_xxx.py and now I would recommend to move this dict outside of the class `GarageDoorOpener`. See the next example for reference:
https://github.com/home-assistant/home-assistant/blob/48402d49dcc54cdba26e44cdfb45e95e6e3edd15/homeassistant/components/homekit/type_locks.py#L15-L20